services/src/Microsoft.Azure.IIoT.Services.Processor.Events/src/Runtime/Config.cs

```diff
@@ -31,7 +31,7 @@ public class Config : DiagnosticsConfig, IEventProcessorConfig, IEventHubConsume
         /// <inheritdoc/>
         public string EventHubPath => _eh.EventHubPath;
         /// <inheritdoc/>
-        public string ConsumerGroup => "events";
+        public string ConsumerGroup => _eh.ConsumerGroupEvents;
```

Comment:
This is where we should put reading the environment variable (see comment above) - also applies to Config.cs below.

Reply:
as mentioned above, I believe this is part of the resource's configuration

Reply:
done